python/cudf/cudf/tests/test_binops.py

```diff
@@ -574,7 +588,6 @@ def test_series_cmpop_mixed_dtype(cmpop, lhs_dtype, rhs_dtype, obj_class):
 )
 def test_series_reflected_ops_scalar(func, dtype, obj_class):
     # create random series
-    np.random.seed(12)
     random_series = utils.gen_rand(dtype, 100, low=10)
```

Comment:
Do we not need to send a seed into `gen_rand`?

Reply:
Done.